src/Root.tsx

```diff
@@ -0,0 +1,22 @@
+import { HashRouter, Navigate, Route, Routes } from 'react-router-dom';
+import { App } from './App';
+import { PeoplePage } from './components/PeoplePage';
+import { HomePage } from './components/HomePage';
+
+export const Root = () => (
+  <HashRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route path="/home" element={<Navigate to="/" replace />} />
```

Comment:
The route path '/home' redirects to '/', which is the same as the root path. This might be redundant unless you have specific reasons for this redirection.